heap_sort/c/heap.c

```diff
@@ -0,0 +1,61 @@
+#include "heap_sort/c/heap.h"
+
+
+
+void add(MinHeap* h, int value) {
+	h->heap[++(h->N)] = value;
+
+	sift(h, h->N);
+	percolate(h, h->N);
+}
+
+void rmv(MinHeap* h, int node) {
+	swap_nodes(h, node, h->N);
+	h->N--;
+
+	sift(h, node);
+	percolate(h, node);
+}
+
+void sift(MinHeap* h, int node) {
+	int son;
+
+	do {
+		son = 0;
+		if(2*node+1 <= h->N) {
+			if (h->heap[2*node+1] < h->heap[2*node]) {
+				son = 2*node+1;
+			} else {
+				son = 2*node;
+			}
+		} else if (2*node <= h->N) {
+			son = 2*node;
+		}
+
+		if(h->heap[node] < h->heap[son]) {
+			son = 0;
+		}
```

Comment:
Missing space before ( in while(  [whitespace/parens] [5]

*Origin: CPPLintBear, Section: `all.cpplint`.*